packages/svelte/svelte.js

```diff
@@ -114,7 +114,7 @@ module.exports = (config = false) => {
                 // Then any remaining bare css.<key> values
                 .replace(
                     new RegExp(`(\\b)css\\.(${selectors})(\\b)`, "gm"),
-                    (match, prefix, key, suffix) => `${prefix}"${exported[key].join(" ")}"${suffix}`
+                    (match, prefix, key, suffix) => (Array.isArray(exported[key]) ? `${prefix}"${exported[key].join(" ")}"${suffix}` : exported[key])
```

Comment:
```suggestion
                    (match, prefix, key, suffix) =>`${prefix}${Array.isArray(exported[key]) ? exported[key].join(" ") : exported[key]}"${suffix}`
```

Instead of forcing the surrounding characters to be quotes use the extracted `prefix` and `suffix` values as the delimiters around the value.

Reply:
I'm noticing that the change you're suggesting drops a quotation mark? previously it's:

```js
`${prefix}"${exported[key].join(" ")}"${suffix}`
```

and afterwards turns into:

```
`${prefix}${Array.isArray(exported[key]) ? exported[key].join(" ") : exported[key]}"${suffix}`
```

In that second example should there not also be qutations between `${prefix}` and the content, a la:

```
`${prefix}"${Array.isArray(exported[key]) ? exported[key].join(" ") : exported[key]}"${suffix}`
```

Reply:
Yes, I wasn't paying enough attention and the quotes need to remain since the injected string won't automatically be wrapped.

Reply:
👍 Sounds good to me!